src/screens/FeedScreen/ContentLink.js

```diff
@@ -1,23 +1,25 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
 import {
   StyleSheet,
   Text,
   TouchableNativeFeedback,
-  View,
-  TouchableWithoutFeedback
+  TouchableWithoutFeedback,
+  View
 } from 'react-native';
 import {useNavigation, useRoute} from '@react-navigation/native';
 
+import BlurredLayer from './elements/BlurredLayer';
 import Card from '../../components/Card/Card';
 import TopicsChip from '../../components/TopicsChip/TopicsChip';
+import dimen from '../../utils/dimen';
+import useCalculationContentLink from './hooks/useCalculatiuonContentLink';
+import useContentFeed from './hooks/useContentFeed';
 import {COLORS} from '../../utils/theme';
+import {DISCOVERY_TAB_USERS} from '../../utils/constants';
 import {fonts, normalize, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {smartRender} from '../../utils/Utils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered and new imports have been added. The addition of `PropTypes` and `moment` is a good practice for type checking and date handling respectively. However, there seems to be a typo in the import statement on line 17. The hook `useCalculatiuonContentLink` should probably be `useCalculationContentLink`. Please correct this.
```diff
- import useCalculationContentLink from './hooks/useCalculatiuonContentLink';
+ import useCalculationContentLink from './hooks/useCalculationContentLink';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->